apps/research-service/src/validations/validators/duplicate-geometry.validator.ts

```diff
@@ -104,7 +106,7 @@ export class DuplicateGeometryValidator implements Validator {
     }
 
     const sitePolygon = await SitePolygon.findOne({
-      where: { siteUuid: siteId, isActive: true },
+      where: { siteUuid: siteId, isActive: true, deletedAt: null },
```

Comment:
Why are you adding `deletedAt` checks here? If the model is marked `paranoid: true`, this is the default behavior.